roboflow/config.py

```diff
@@ -1,13 +1,23 @@
 import os
 
+CLASSIFICATION_MODEL = os.getenv("CLASSIFICATION_MODEL", default="ClassificationModel")
+INSTANCE_SEGMENTATION_MODEL = "InstanceSegmentationModel"
```

Comment:
i didn't allow this to be overridden from the `env` because i didn't understand the use-case. i'm happy to make that change if it makes sense tho.